build.gradle

```diff
@@ -233,9 +234,11 @@ distributions {
 
 docker {
   javaApplication {
-    baseImage = 'openjdk:8-jre-slim'
+    // https://bmuschko.github.io/gradle-docker-plugin/current/user-guide/#extension_2
+    // update base image to openjdk 17 jdk slim. other jdk like termurin, etc generates error
+    baseImage = 'eclipse-temurin:17-jre-alpine'
```

Comment:
>  Was there a particular reason to switch to the -alpine tag or could we use 17-jre-jammy instead?

There was one reason to not use `jre-slim` because docker tasks were not building successfully due to some missing dependencies. So the first try I made was with `alpine` which worked and then it was decided on.
 
I wasn't aware of the fact that it could have troubles with derived images expecting Ubuntu. 

I now tested with `jre-jammy` and the task builds. I will create a fixup PR for the same. 

Thank you for the feedback @florianesser and apologies for the inconvenience.